src/main/kotlin/org/jitsi/jicofo/xmpp/XmppConfig.kt

```diff
@@ -212,6 +212,9 @@ data class XmppComponentConfig(
     val port: Int = -1,
     val secret: String = ""
 ) {
+    val enabled: Boolean
```

Comment:
nit: since `hostname` is a val, this could be: `val enabled: Boolean = hostname != ""`  There's also `isEmpty` or `isBlank` helpers for `String`, depending on what we actually care about hostname being.